Update Ionic 3 quickstart [QS-383] (#8480)
auth0/docs: articles/quickstart/native/ionic3/01-login.md

YOUR_PACKAGE_ID://${account.namespace}/cordova/YOUR_PACKAGE_ID/callback
```

> To be able to make requests from your application to Auth0. Set the following origins in your [Application Settings](${manage_url}/#/applications/${account.clientId}/settings).

Reviewer: ```suggestion
To be able to make requests from your application to Auth0, set the following origins in your [Application Settings](${manage_url}/#/applications/${account.clientId}/settings).
```

```bash
http://localhost, ionic://localhost, http://localhost:8100
```

> the origins `http://localhost` and `ionic://localhost` are needed for Android and iOS respectively, and `http://localhost:8100` is needed you're running your application with `livereaload` option.

Reviewer: ```suggestion
The origins `http://localhost` and `ionic://localhost` are needed for Android and iOS. respectively The origin `http://localhost:8100` is needed when you are running the application with `livereaload` activated.
```
